components/builder-originsrv/src/migrations/origin_public_keys.rs

```diff
@@ -83,11 +83,24 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         RETURN;
                     END
                     $$ LANGUAGE plpgsql STABLE"#)?;
-
-    migrator.migrate("originsrv",
-                     r#"ALTER TABLE origin_public_keys
+    migrator
+        .migrate("originsrv",
+                 r#"ALTER TABLE origin_public_keys
                         DROP CONSTRAINT IF EXISTS
                           origin_public_keys_full_name_key"#)?;
-
+    migrator
+        .migrate("originsrv-2",
```

Comment:
I can't see how to do transactions inside the builder migrations.  The migrator runs every migration in its own transaction automatically, which means you can't have multiple statements inside a migration or you get an error that says `cannot insert multiple commands into a prepared statement`.  It doesn't look like from the migrator code that there's a way to do transactions manually.

If someone knows how to do transactions with multiple statements inside the migrator, please chime in.  Otherwise, maybe it's ok to leave this as is?  The odds of a dupe sneaking in between when these two migrations run seems low.